tests/postgres/migrations_split/0_create_index.sql

```diff
@@ -0,0 +1,8 @@
+-- no-transaction
+
+CREATE TABLE test_table (x int);
+-- split-migration
+CREATE INDEX CONCURRENTLY test_table_x_idx ON test_table (x);
+-- split-migration
+INSERT INTO test_table (x) VALUES (1);
+-- prove that you can have a comment that won't split -- split-migration DROP TABLE does_not_exist;
```

Comment:
I confirmed that if I remove the `\n`s from the string split this gets split such that `DROP TABLE does_not_exist;` becomes runnable SQL that then fails to run.